docs/book/src/tasks/bootstrap/kubeadm-bootstrap/kubelet-config.md

```diff
@@ -0,0 +1,354 @@
+# Kubelet Configuration
+
+CAPBK has several ways to configure kubelet.
+
+- [Pass `KubeletConfiguration` file via `KubeadmConfigSpec.files`](#pass-kubeletconfiguration-file-via-kubeadmconfigspecfiles)
+- [Set kubelet flags via `KubeadmConfigSpec.kubeletExtraArgs`](#set-kubelet-flags-via-kubeadmconfigspeckubeletextraargs)
+- [`kubeletconfiguration` patch target](#use-the-kubeletconfiguration-patch-target)
+
+## Pass `KubeletConfiguration` file via `KubeadmConfigSpec.files`
+
+You can use `KubeadmConfigSpec.files` to put any files on nodes. This example puts a `KubeletConfiguration` file on nodes via `KubeadmConfigSpec.files`, and makes kubelet use it via `KubeadmConfigSpec.kubeletExtraArgs`. You can check available configurations of `KubeletConfiguration` on [Kubelet Configuration (v1beta1) | Kubernetes](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration).
+
+This method is easy to replace the whole kubelet configuration, but it is not easy to replace only a part of the kubelet configuration.
```

Comment:
Q: which kubelet configuration it is replacing? I assume the one generated by kubeadm, might be worth to make it explicit

Reply:
3463620 fixed to make it clear where the default kubelet configuration comes from.